tools/bumper/bumper.go

```diff
@@ -80,18 +87,23 @@ func main() {
 
 			// Create PR
 			exitWithError(fmt.Errorf("create PR not implemented yet"))
+			// update components yaml in the bumping repo instance
+			componentsConfig, err = parseComponentsYaml(configPath)
+			if err != nil {
+				exitWithError(errors.Wrap(err, "Failed to parse components yaml during bump"))
+			}
 
 			// update component's entry in config yaml
 			component.Commit = updatedReleaseCommit
 			component.Metadata = updatedReleaseTag
 			componentsConfig.Components[componentName] = component
-			err = updateComponentsYaml(inputArgs.componentsConfigPath, componentsConfig)
+			err = updateComponentsYaml(configPath, componentsConfig)
 			if err != nil {
 				exitWithError(errors.Wrap(err, "Failed to update components yaml"))
 			}
 
 			logger.Printf("Running bump-%s script", componentName)
-			cmd := exec.Command("make", fmt.Sprintf("bump-%s", componentName))
+			cmd := exec.Command("make", "-C", bumpingRepo.gitRepo.localDir, fmt.Sprintf("bump-%s", componentName))
```

Comment:
This should be encapsulated at cnaoRepo `cnaoRepo.Bump(componentName)`

Reply:
Done